metrics/statsd.go

```diff
@@ -12,10 +12,11 @@ import (
 	"github.com/odpf/meteor/recipe"
 )
 
-var (
+const (
 	runDurationMetricName    = "runDuration"
 	runRecordCountMetricName = "runRecordCount"
 	runMetricName            = "run"
+	sinkMetricName           = "sinkStatus"
```

Comment:
let's generalise it to plugin and keep same format like run recipe
```suggestion
	pluginRunMetricName           = "runPlugin"
```